platform/telemetry/src/TelemetryCapability.js

```diff
@@ -300,7 +300,7 @@ define(
             var defaultRange = metadata.valuesForHints(['range'])[0];
             defaultRange = defaultRange ? defaultRange.key : undefined;
 
-            var sourceMap = _.indexBy(metadata.values(), 'key');
+            var sourceMap = _.keyBy(metadata.values(), 'key');
```

Comment:
Let's re-implement natively with map

Reply:
Done, but not with map(). Used reduce().